cmd/swagger/commands/generate/support.go

```diff
@@ -42,8 +42,11 @@ func (s *Support) getOpts() (*generator.GenOpts, error) {
 		Principal:     s.Principal,
 		DumpData:      s.DumpData,
 		DefaultScheme: s.DefaultScheme,
+		Template:      s.Template,
 		TemplateDir:   string(s.TemplateDir),
-	}, nil
+	}
+	contribOptionsOverride(opts)
```

Comment:
may be guarded by `!= "" or any default value here.

Reply:
@fredbi what do you mean here?

Reply:
Shouldn't we avoid calling when not used?
```go
if Template != "" {
  contribOptionsOverride(opts)
}
````

Reply:
Oh yes I did not figure it out. This is not very important